stacks/jupyterhub-pyspark-hdfs/spark_connect.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: spark-connect-log-config
+data:
+  log4j2.properties: |-
+    appenders = CONSOLE
+
+    appender.CONSOLE.type = Console
+    appender.CONSOLE.name = CONSOLE
+    appender.CONSOLE.target = SYSTEM_ERR
+    appender.CONSOLE.layout.type = PatternLayout
+    appender.CONSOLE.layout.pattern = %d{ISO8601} %p [%t] %c - %m%n
+    appender.CONSOLE.filter.threshold.type = ThresholdFilter
+    appender.CONSOLE.filter.threshold.level = DEBUG
+
+    rootLogger.level=INFO
+    rootLogger.appenderRefs = CONSOLE
+    rootLogger.appenderRef.CONSOLE.ref = CONSOLE
+
+---
+apiVersion: spark.stackable.tech/v1alpha1
+kind: SparkConnectServer
+metadata:
+  name: spark-connect
+  labels:
+    stackable.tech/vendor: Stackable
+spec:
+  image:
+    # Using an image that includes scikit-learn (among other things)
+    # because this package needs to be available on the executors.
+    custom: oci.stackable.tech/sdp/spark-connect-client:3.5.5-stackable0.0.0-dev
```

Comment:
This might change depending on the outcome mentioned in the dependent PR:

https://github.com/stackabletech/docker-images/pull/1071#issuecomment-2847048547